docs/AddUnitTests.md

```diff
@@ -0,0 +1,62 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+
+## Unit Testing
+
+We use the [MSTest framework](https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.testtools.unittesting?view=mstest-net-1.2.0) to write unit tests.
+
+### Unit Test Setup
+Setting up unit tests for an assembly is trivial. 
+The steps are:
+1. Use Visual Studio to **add a new Unit Test Project** (available as a new project template listed under Visual C#, then Test)
+   1. In Visual Studio, right-click on the solution and add a new project.
+   2. Select unit tests from the template list.
+   3. Name your project consistently with the existing test projects. If your project is Widgets.Stuff, then your test project should be named *Widgets.Stuff**Tests***.
+   4. Finish setting up the project by following instructions in the [Readme.md](../src/Readme.md)
+
+2. **Add a unit test class** to the project. Name the class after the class it will be testing--if the class is named *ExplodingWidget*, then the test class should be named *ExplodingWidget**Tests***
+    1. In Visual Studio, right-click on the test project and add a new test class.
+    2. Rename the test class from UnitTests to the name that matches your class name. So if your class is SuperList, your unit test class should be SuperListTests.
+
+>Hint: if you right-click on a method that you need to test, you'll get a template that does this for you automatically. The only thing it doesn't do is to name the actual test according to what's described in Standards
+
+For each test, add a method like this:
+
+```C#
```

Comment:
>C# [](start = 3, length = 2)

Not sure what the "C#" is supposed to do here. It gets ignored in the page rendering (see [https://github.com/microsoft/axe-windows/blob/61cea32fd86e572cf6b753021b441e847324ee7d/docs/AddUnitTests.md](https://github.com/microsoft/axe-windows/blob/61cea32fd86e572cf6b753021b441e847324ee7d/docs/AddUnitTests.md) for how the page renders)

Reply:
It's just a way of telling the markup what sort of language the code block contains I think.

Reply:
Neat trick--I hadn't come across that before :)

---
In reply to: [299143281](https://github.com/microsoft/axe-windows/pull/88#discussion_r299143281) [](ancestors = 299143281)